improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -314,10 +314,11 @@ def process(
         self,
         forecast_at_percentiles: Cube,
         no_of_percentiles: Optional[int] = None,
-        sampling: str = "quantile",
+        sampling: Optional[str] = "quantile",
+        percentiles: Optional[List] = None,
```

Comment:
As this stands I can happily request any arbitrary set of percentiles (e.g -10th percentile, along with the 110th percentile). No warning or error is raised at any step in the process, with the resulting values (at least in my test) simply being the ecc bounds. The resulting cube has an amusing coordinate:

```
DimCoord(array([-10.,  30.,  70., 110.], dtype=float32), standard_name=None, units=Unit('%'), long_name='percentile')
```
Please can you put in some basic checking that the percentile values requested are sensible, otherwise a misplaced 0 or minus sign could be a bit confusing.

Reply:
I've added a check for unexpected percentile values.